src/main/java/org/kiwiproject/dropwizard/util/admin/StandardAdminConfigurator.java

```diff
@@ -0,0 +1,117 @@
+package org.kiwiproject.dropwizard.util.admin;
+
+import static java.util.Objects.isNull;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.kiwiproject.base.KiwiPreconditions.requireNotNull;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.health.HealthCheckRegistry;
+import com.google.common.annotations.VisibleForTesting;
+import io.dropwizard.setup.Environment;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import org.kiwiproject.config.TlsContextConfiguration;
+import org.kiwiproject.dropwizard.util.config.KeystoreConfig;
+import org.kiwiproject.dropwizard.util.health.HttpConnectionsHealthCheck;
+import org.kiwiproject.dropwizard.util.health.keystore.ExpiringKeystoreHealthCheck;
+import org.kiwiproject.dropwizard.util.metrics.ServerLoadGauge;
+import org.kiwiproject.dropwizard.util.task.ServerLoadTask;
+
+import java.util.stream.Stream;
+
+/**
+ * Set of utilities to assist in configuring administrative functions of a Dropwizard service.
```

Comment:
Perhaps we add second paragraph explaining that "standard" is only from the point of view of this library.

Or, we could consider dropping "Standard" from this (and possibly other) class names, though I cannot think of a good name and just `AdminConfigurator` might be a bit too vague. Not sure...